Code/GraphMol/MarvinParse/MarvinParser.cpp

```diff
@@ -233,7 +236,7 @@ class MarvinCMLReader {
       }
 
       return res;
-    } catch (const std::exception &e) {
+    } catch (const std::exception &) {
       delete res;
       res = nullptr;
```

Comment:
I'm definitely a member of the "set the pointer to null immediately after freeing it, even if you don't have to, because you never know what changes someone might make in 5 years." school of thought